nilearn/datasets/atlas.py

```diff
@@ -624,17 +636,22 @@ def fetch_atlas_aal_spm_12(data_dir=None, url=None, resume=True, verbose=1):
 
     Licence: unknown.
     """
-    spm_version = 12
+    versions = ['SPM5', 'SPM8', 'SPM12']
+    if version not in versions:
+        raise ValueError('The version of AAL requested "%s" does not exist.'
+                         'Please choose one among %s.' %
+                         (version, str(versions)))
+
     if url is None:
-        baseurl = "http://www.gin.cnrs.fr/AAL_files/aal_for_SPM%i.tar.gz"
-        url = baseurl % spm_version
+        baseurl = "http://www.gin.cnrs.fr/AAL_files/aal_for_%s.tar.gz"
+        url = baseurl % version
     opts = {'uncompress': True}
 
-    dataset_name = "aal_spm_12"
+    dataset_name = "aal_" + version
```

Comment:
Stupid question: would the description not be useful irrespective of the SPM version ?